source/blueprints/user/routes.py

```diff
@@ -121,3 +122,32 @@ def change_password():
         },
     ).json()
 
+
+@user_bp.route("/delete_account", methods=["DELETE"]) 
+@jwt_required()
+def user_del():
```

Comment:
Juuust another really small tweak here too, prefer to use descriptive names for the functions:
`def delete_user()` it's likely cleaner and more readable than names such as `user_del` or `del_user`.